db-scripts/imsdb3.sql

```diff
@@ -2,9 +2,9 @@ CREATE TABLE public.ims_internship_evaluate_form (
     id bigserial NOT NULL,
     "name" varchar NOT NULL,
     surname varchar NOT NULL,
-    file_path varchar NOT NULL,
+    file_content bytea NOT NULL,
```

Comment:
Kim, nasıl, ne zaman ve nedne buna karar verdi? Veritabanında binary dosya tutmak gibi bir talebimiz bulunmuyor.

Reply:
@ykartal hocam createIntershipEvaluateForm kısmında dosyada yüklenmesi gerekir dediniz, haklısınız farkedememişim. Ama beceremedim, yetkili o ekranı nasıl görür http den ne isteriz vs kestiremedim. Uğraşıp araştırırken file için byte olarak tutulması daha doğru olur minvalinde yazılar okudum,  ben de  değiştirdim.  Geri eski haline çeviririm.

Reply:
Veritabanında tutulan binary dosyalar performans, saklama ve yedekleme problemlerine sebep olabildiğinden tercih etmiyoruz.

Reply:
Dosya olarak gelen veriyi diskte bir yere kaydedip ilgili yolu veritabanına kaydetmemiz en doğrusu

`@RequestPart(value = "file", required = true) MultipartFile file`

dosya yükleme konusunda bir fikir verebilir.

Reply:
@ykartal Tamam hocam teşekkür ederim, bu şekilde yazıyorum, ancak String filePath = "C:/uploads/" + fileName; burda pathi ne verebilirim? String filePath = "src/main/resources/static/uploads/" + fileName; olur mu mesela ?

Reply:
Dosya yüklenecek şekilde yazdım hocam, ancak String filePath = "C:/uploads/" + fileName; burda pathi ne verebilirim?